### PR TITLE
fix regex expression for stat_prefix

### PR DESCRIPTION
**Pull request by varks**
fixes regex expression for stat_prefix added in https://github.com/istio/istio/pull/40503

### CODE REVIEW

File: pkg/bootstrap/config.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ const (
 	requiredEnvoyStatsMatcherInclusionSuffixes = rbacEnvoyStatsMatcherInclusionSuffix + ",downstream_cx_active" // Needed for draining.
 
 	// required for metrics based on stat_prefix in virtual service.
-	requiredEnvoyStatsMatcherInclusionRegexes = "vhost.*.route.*"
+	requiredEnvoyStatsMatcherInclusionRegexes = "vhost\\.*\\.route\\.*"
```
Comment (howardjohn) [inline pkg/bootstrap/config.go:64]: nit: use backtick instead of " to avoid needing double escapes

**Review by howardjohn — APPROVED** [review]

File: pkg/bootstrap/config.go (hunk 1 above)
Comment (varks) [reply pkg/bootstrap/config.go]: ack, fixed it.

**Comment by varks** [comment]
@howardjohn when you get a chance, could you add the `ok to test` label. thank you.

**Comment by zirain** [comment]
can you add a e2e tests for this?

**Pull request merged**